src/libs/Navigation/switchPolicyID.ts

```diff
@@ -93,8 +102,10 @@ export default function switchPolicyID(navigation: NavigationContainerRef<RootSt
         return;
     }
 
+    const shouldPushCentralPane = !getIsNarrowLayout() || shouldOpenSearchPage;
```

Comment:
I find it hard to understand what does "push" mean in this context. If you think this is obvious to people working with navigation then feel free to leave as is.
If not then perhaps some other name or a comment?